cmd/troubleshoot/cli/run.go

```diff
@@ -210,6 +210,7 @@ func runCollectors(v *viper.Viper, collector troubleshootv1beta1.Collector, prog
 	collectSpecs = append(collectSpecs, collector.Spec.Collectors...)
 	collectSpecs = ensureCollectorInList(collectSpecs, troubleshootv1beta1.Collect{ClusterInfo: &troubleshootv1beta1.ClusterInfo{}})
 	collectSpecs = ensureCollectorInList(collectSpecs, troubleshootv1beta1.Collect{ClusterResources: &troubleshootv1beta1.ClusterResources{}})
+	collectSpecs = ensureCollectorInList(collectSpecs, troubleshootv1beta1.Collect{Rook: &troubleshootv1beta1.Rook{}})
```

Comment:
What happens if Rook isn't installed?

Reply:
If no pods exist under the 'root-ceph' namespace, then the command won't run and no status files will get created.